src/Momento.Sdk/ISimpleCacheClient.cs

```diff
@@ -86,7 +86,7 @@ public interface ISimpleCacheClient : IDisposable
     /// <param name="cacheName">Name of the cache to store the item in.</param>
     /// <param name="key">The key to set.</param>
     /// <param name="value">The value to be stored.</param>
-    /// <param name="ttlSeconds">TTL for the item in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL.</param>
+    /// <param name="ttl">TTL for the item in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL. If specified must be strictly positive.</param>
```

Comment:
is this comment still relevant?  Can TimeSpans be negative?

Reply:
Yes [they can be negative](https://learn.microsoft.com/en-us/dotnet/api/system.timespan?view=net-7.0), eg they can represent the 24 hours ago relative to an index time.